api/app/tests/apps/signals/test_tasks.py

```diff
@@ -138,3 +141,216 @@ def test_translation_skipped_category_not_in_translations_split(self):
 
         cats = CategoryAssignment.objects.filter(_signal=child_signal)
         self.assertEqual(cats.count(), 1)
+
+
+class TestTaskUpdateStatusChildrenBasedOnParent(TransactionTestCase):
+    def setUp(self):
+        self.parent_signal = factories.SignalFactory.create()
+        self.child_signal_1, self.child_signal_2 = factories.SignalFactory.create_batch(2, parent=self.parent_signal)
```

Comment:
Consider setting the state explicitly here, so that you don't have to lookup the `SignalFactory` behavior with respect to `signal.status.state`